build/DependencyVersions.props

```diff
@@ -32,9 +32,16 @@
     <CliCommandLineParserVersion>0.1.1-alpha-174</CliCommandLineParserVersion>
     <CliMigrateVersion>1.2.1-alpha-002133</CliMigrateVersion>
     <MicroBuildVersion>0.2.0</MicroBuildVersion>
-    <SpaTemplateVersion>1.0.417</SpaTemplateVersion>
     <XliffTasksVersion>0.2.0-beta-000042</XliffTasksVersion>
     <DotNetCliArchiverVersion>0.2.0-beta-000059</DotNetCliArchiverVersion>
+    <AspNetCoreTemplatePackageVersion>2.0.3</AspNetCoreTemplatePackageVersion>
```

Comment:
@natemcmaster could you please review this merge? It is merging back release/2.0.0 into master, through release/15.5

Reply:
aspnet changes look fine.